### ISSUE TITLE
"Wrong parity" when using local plate-solving

### ISSUE THREAD

**Issue by QK-sampson** [ThemosTsikas/PhotoPolarAlign#10]
Hi Themos,
The weather is not allow me to use your great PPA so I test it using your photo v.jpg and h.jpg. 
When I tried to find polar axis using local plate-solving (astrometry.net under my Raspi), it indicated "wrong parity" but it was fine when using on-line plate-solving.
By the way, it was fine for using other "horiz" and "vert". I don't know how to solve this problem.
Thanks in advance =)

Sampson

**Comment by QK-sampson** [comment]
Hi Theoms,
I found that using index-4112.fits to solve the both images "v.jpg" and "h.jpg" would cause wrong parity. If I remove index-4112.fits and use 4113.fits for v.jpg and  4115.fits for "h.jpg" would be fine.

I found the solved result of h.jpg would be "Field rotation angle: up is -177.957 degrees E of N" using index-4112.fits but it would be "Field rotation angle: up is 107.546 degrees E of N" using 4113.fits.
So what index files you would recommend to use?

 Thanks in advance =)

Sampson
